bindings/java/src/main/native/Makefile.am

```diff
@@ -11,6 +11,7 @@ java_build_dir = $(builddir)/build-java
 maven_repo    ?= $(java_build_dir)/.deps
 jarfile        = $(java_build_dir)/jucx-@VERSION@.jar
 javadir        = $(top_srcdir)/bindings/java
+objdir         = $(builddir).libs
```

Comment:
objdir is defined by configure so it should not be redefined manually.. 
see https://github.com/openucx/ucx/blob/4cbf3246490d283f87981328c4da1d3d99c489e3/configure.ac#L140